data/compatibility/supportStatus.yml

```diff
@@ -5,13 +5,14 @@
   supported: "No, development only"
   releaseDate:
   eolDate:
-  k8sVersions: ["1.25", "1.26", "1.27", "1.28"]
+  k8sVersions: ["1.25", "1.26", "1.27", "1.28", "1.29"]
   testedK8sVersions: ["1.23", "1.24"]
 - version: "1.20"
   supported: "Yes"
   releaseDate: "Nov 14, 2023"
   eolDate: "~Jul 2024 (Expected)"
-  k8sVersions: ["1.25", "1.26", "1.27", "1.28"]
+  
+  k8sVersions: ["1.25", "1.26", "1.27", "1.28", "1.29"]
```

Comment:
I think this doesn't seem correct to me. 1.20 has announced support for k8s 1.25 to 1.28, and this will lead to 1.20 supporting five k8s versions.

Reply:
See https://deploy-preview-14486--preliminary-istio.netlify.app/latest/docs/releases/supported-releases/#support-status-of-istio-releases

Reply:
Do we have a limit to number of k8s versions we support? Even though k8s 1.25 has now hit EOL, it doesn't feel like we can remove it from the table. The difference in k8s releases policy means that we'll soon have a version of k8s that doesn't have a supported Istio version. Supporting 5 versions for a bit seems preferable to that outcome

Reply:
We usually support four versions. If 1.29 is listed in Istio 1.20, it might lead to the misconception that Istio 1.20.0 - 1.20.2 also support Kubernetes 1.29, but they have not been tested. Therefore, I think for Istio 1.20,  we will stick to 1.25-1.28, and the upcoming Istio 1.21 is likely to support 1.26-1.29. I think @ericvn knows much about this.

Reply:
1.20.2 has had acceptance tests run on k8s 1.29: https://istio.slack.com/archives/C05SW44NP32/p1704400785770819

We're going to run into this every year because we have 4 releases and k8s only has 3. Istio 1.21 won't be released until 2/19 (assuming no delays) while k8s 1.29 was released 12/20/2023. The holidays help us out a bit but still we need to have an Istio version out sooner that supports 1.29 IMO

Reply:
Yep makes sense to me 👍🏾

Reply:
Good to know this, thanks!